Dockerfile.build

```diff
@@ -1,8 +1,16 @@
 FROM golang:1.18
 
+ENV GOOS="linux"
```

Comment:
I would make these into build args with defaults, so that you could use the image to test multiple archs

Reply:
i'm not sure i understand what you mean..